bot/exts/info/pypi.py

```diff
@@ -63,6 +72,10 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
                     else:
                         embed.description = "No summary provided."
 
+                    upload_date = _get_latest_distribution_timestamp(response_json)
+                    if upload_date:
+                        embed.set_footer(text=f"Uploaded to PyPI on {upload_date}")
```

Comment:
```suggestion
                        embed.set_footer(text=f"Latest release uploaded to PyPI on {upload_date}")
```
Thoughts on this to remove ambiguity of it referring to first release? Or do you think it's obvious?

Reply:
Why not more concisely `Released on {date}`? It should be reasonably obvious that this represents the time when the "current upload" of that version was published. After all, this is what's displayed on the pypi website:

```suggestion
                        embed.set_footer(text=f"Released on {upload_date}")
```

![image](https://github.com/python-discord/bot/assets/44247924/6c76308b-de40-40dd-a29a-b66f6c9bbb50)

Reply:
And since we're modifying the text I'd like to add that explicitly stating "to PyPI" is somewhat unnecessary since it's a PyPI command. @GabeMillikan's suggestion also sounds reasonable to me.

Reply:
I'm happy with the footer text to be just a Released / Uploaded + the date. However I do like to say that I added the "Uploaded to PyPI" after previous comments on that someone said it was unclear as to what it was uploaded. I do agree however that it's enough to just state "Released on <date>", so I will revert it back to that.